.npmignore

```diff
@@ -1,7 +1,19 @@
 # OSX
-.DS_Store
-
 /*
 !lib/
-!app/lib
-!bin
+!app/lib/
+!icon-scripts
+.DS_Store
+.eslintrc.yml
+src/
+app/src/
+app/node_modules
+*tsconfig.tsbuildinfo
+*package-lock.json
+*tsconfig.json
+*jestSetupFiles*
+*-test.js
+*-test.js.map
+*.test.d.ts
+*.test.js
+*.test.js.map
```

Comment:
> It's been a long time since I've worked with npm, does `/*` not cover this?

@jiahaog I thought too! I'm not sure exactly of the logic and I haven't looked at the code for `npm pack`, but I manually inspected the generated tarfile, and still ended up with excess files. This .npmignore is the result of fiddling with it to make the artifact optimally small.